bindings/cs/rl.net/LiveModel.cs

```diff
@@ -899,6 +964,33 @@ public MultiSlotResponse RequestMultiSlotDecision(string eventId, string context
             return result;
         }
 
+        public bool TryRequestMultiSlotDecision(string eventId, string contextJson, ActionFlags flags, out MultiSlotResponse response, int[] baselineActions, ApiStatus apiStatus)
+        {
+            response = new MultiSlotResponse();
+
+            GC.KeepAlive(this);
+            return this.TryRequestMultiSlotDecision(eventId, contextJson, flags, response, baselineActions, apiStatus);
+        }
+
+        public bool TryRequestMultiSlotDecision(string eventId, string contextJson, ActionFlags flags, MultiSlotResponse response, int[] baselineActions, ApiStatus apiStatus)
+        {
+            int result = LiveModelRequestMultiSlotDecisionWithBaselineAndFlags(this.DangerousGetHandle(), eventId, contextJson, (uint)flags, response.DangerousGetHandle(), baselineActions, apiStatus.ToNativeHandleOrNullptrDangerous());
```

Comment:
You should add `GC.KeepAlive(this)` (and maybe `GC.KeepAlive(response)` before the return, though that is probably not necessary, since it should be on the stack in this call), since otherwise the GC might decide to deallocate the object while the request is in flight.